docs/en/query_language/functions/date_time_functions.md

```diff
@@ -169,6 +169,77 @@ Converts a date or date with time to a UInt16 number containing the ISO Year num
 
 Converts a date or date with time to a UInt8 number containing the ISO Week number.
 
+## toCustomYear
+
+Converts a date or date with time to a UInt16 number containing the Custom Year number.
+```
+toCustomYear(DateTime, [, CustomDate][, Timezone])
```

Comment:
It seems your PR doesn't contain the function `toCustomYear` anymore.

Reply:
Yes, it needs to be deleted.